_posts/2015-06-07-0.37-release.md

```diff
@@ -0,0 +1,36 @@
+---
+title:  "Release 0.37"
+author: Julien Pagès
+---
+
+I'm excited to announce that mozregression now support **using taskcluster**
+**for inbound bisection** with this release!
+
+This is the major improvement, and this speed up inbound hunting (Yeah!).
+And mozregression made a first step to handle the migration from [ftp to s3]!
```

Comment:
Would probably reword this paragraph to something like:

"This considerably speeds up the process of downloading builds from integration branches (like mozilla-inbound) and also supports work by Mozilla Release Engineering to migrate build storage from ftp to s3."